bats/tests/helpers/vm.bash

```diff
@@ -10,18 +10,18 @@ wait_for_shell() {
 }
 
 factory_reset() {
-    rdctl factory-reset
```

Comment:
We should at least keep a `rdctl shutdown` or something? Better to gracefully shut down than murder things.

Reply:
I don't know if this will work or not, but it doesn't seem to be necessary. This can be worked on in the PR to make support for bats with npm more robust.

Reply:
I've checked with @ericpromislow that `rdctl shutdown` will time out after 30s, and then uses `pkill` to shutdown the processes (which won't work for `npm run dev`). So it won't block.

I've added it into the `using_npm_run_dev` block before running `pkill`. There should be no reason to run `shutdown` before `factory-reset` for the non-npm case.